src/win/util.c

```diff
@@ -1778,3 +1778,20 @@ int uv_os_uname(uv_utsname_t* buffer) {
   buffer->machine[0] = '\0';
   return r;
 }
+
+int uv_gettimeofday(uv_timeval_t* tv) {
+  /* Based on https://doxygen.postgresql.org/gettimeofday_8c_source.html */
+  const uint64_t epoch = (uint64_t) 116444736000000000ULL;
```

Comment:
Shouldn't that be `int64_t` because QuadPart is signed?

Reply:
@bnoordhuis I thought `QuadPart` was unsigned in `ULARGE_INTEGER`. (https://msdn.microsoft.com/en-us/windows/desktop/aa383742, https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/c57d9fba-12ef-4853-b0d5-a6f472b50388).

Reply:
Also @bnoordhuis, is there any reason you know of not to use something similar to what @refack posted in https://github.com/libuv/libuv/pull/2221/files#r266700428? It certainly seems simpler, so I'd prefer it if possible.

Reply:
I guess thread safety would be at least one reason.

Reply:
> I thought QuadPart was unsigned in ULARGE_INTEGER. 

You're right and I don't know why I thought that... Objection withdrawn.